eth/auction.go

```diff
@@ -990,3 +996,135 @@ func (c *AuctionClient) AuctionEventsByBlock(blockNum int64,
 	}
 	return &auctionEvents, nil
 }
+
+// GetCoordinatorsLibP2PAddrs return the libp2p addr associated to each coordinator
+// that has been registered so far
+func (c AuctionClient) GetCoordinatorsLibP2PAddrs() ([]multiaddr.Multiaddr, error) {
+	// Get events
+	query := ethereum.FilterQuery{
+		Addresses: []ethCommon.Address{
+			c.address,
+		},
+		Topics: [][]ethCommon.Hash{{logAuctionSetCoordinator}},
+	}
+	logs, err := c.client.client.FilterLogs(context.TODO(), query)
```

Comment:
I believe `client.client` looks like a code smell. I'd recommend to use names that specify what the client is for.

Reply:
I don't like this either, but this comes from the underlying struct `AuctionClient` that has +30 methods. Changing this would require a considerable amount of changes.

What do you think about opening an issue to address this, and move on with this PR?

Also, I believe that the other SC related structs follow the same pattern, so we could take advantage of a dedicated issue and improve all of them at once

Reply:
Issue created for handle this: https://github.com/hermeznetwork/hermez-node/issues/1063

Reply:
So, it's event worst because the call is: client.client.client

Please, open an issue asking to change the client variable name showing what's this client for. Whether it's for Ethereum it needs to have AuctionEthClient for instance. Please try to be very specific in the issue's text to avoid misunderstand. Thanks.